app/utils/whitelabel.ts

```diff
@@ -33,9 +33,10 @@ interface WhiteLabelSite {
 
 // Include a domain in config.js
 function determineWhiteLabelSite(): WhiteLabelSiteKey {
-  const subdomain = window.location.host.split('.')[0];
+  const domain = window.location.host;
+  const subdomain = domain.split('.')[0];
   if (subdomain === String(config.SFFAMILIES_DOMAIN) || subdomain === `${String(config.SFFAMILIES_DOMAIN)}-staging`) return 'SFFamilies';
-  if (subdomain === String(config.MOHCD_DOMAIN) || subdomain === `${String(config.MOHCD_DOMAIN)}-staging`) return 'SFServiceGuide';
+  if (subdomain === String(config.MOHCD_DOMAIN) || domain === `staging.${String(config.MOHCD_DOMAIN)}.org`) return 'SFServiceGuide';
```

Comment:
I don't think this will work in production, where the url for sfserviceguide is www.sfserviceguide.org.

Reply:
Confirmed will work in production.